files/initramfs-tools/varlog

```diff
@@ -17,6 +17,9 @@ for x in "$@"; do
     case "$x" in
         varlog_size=*)
             varlog_size="${x#varlog_size=}"
+	;;
+	*mmc_host*)
```

Comment:
So this change depends on a "*mmc_host*" kernel command line parameter. I agree with your description that this is a crude check. Is there a way for a vendor to override this for their platform (or for an end-user building an image, for that matter) in the case it is not desired? What is populating the "*mmc_host*" kernel command line parameter?

Reply:
On my device the actual kernel command line parameter is:
block_flash=pci0000:00/0000:00:14.7/mmc_host/.*$
instead of looking for *block_flash* I'm looking for an actual *MMC* discriminator hence the \*mmc_host\*
again this may need more research to find a robust way of detecting that the image depends on flash and its not just a usb key with a eMMC component that is inserted and not used
Of all the platforms I have in my lab ( 6+ ) this is the only one I have found this.

Reply:
as for your question on overriding: maybe introduce a rules/config variable and check on that?
our vendor's recommendation for this change makes it clear that there is a real danger for the longevity of the switch if the default behavior is left as-is